apps/hash-frontend/src/graphql/queries/user.queries.ts

```diff
@@ -8,11 +8,14 @@ export const isShortnameTaken = gql`
   }
 `;
 
+// We follow links rightwards twice in order to get the orgs that the user is a member of, and the orgs' avatars
+// This may lead to significant overfetching when the user has a lot of e.g. pages
+// @todo consider fetching the org data separately (e.g. in useOrgs)
```

Comment:
Not suggesting to do it now, just wondering how much work this would be? Fetching at a depth of two seems bad (most likely not an issue in the short-term future)

Reply:
Not much – tracking in H-657. Ideally we'd be able to filter the types of links which are followed by that's much more work